lib/queries/get-challenges-by-tag.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
This function would work but I'm not sure it belongs to the tags plugin. The implementation is very specific to the structure of the challenges table so I think it should belongs to the challenges plugin. Another solution is to keep the more abstract logic function getByTag in the tags plugin by passing the name of the columns as a parameter to the function, this will allow to keep the implementation of the function abstract enough, see https://github.com/postgres-plugin/tags-system/issues/52#issuecomment-261768649

So I think we need to choose between:
- Create less abstract functions in their own plugins : getChallengesByTag, getOrganisationByTag,...
- Create a more abstract function getByTag which belongs to the tag plugin.
@JMurphyWeb what is your preference?